### PR TITLE
bug fixes + updates

### PR DESCRIPTION
**Pull request by Mehrad0711**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,9 +5,15 @@ MAINTAINER Thingpedia Admins <thingpedia-admins@lists.stanford.edu>
...
 WORKDIR /opt/genienlp/
-ARG DECANLP_VERSION=master
+ARG GENIENLP_VERSION=master
 RUN pip3 install --upgrade pip
-RUN git fetch && git checkout ${DECANLP_VERSION} && pip3 install -e .
+RUN git fetch && git checkout ${GENIENLP_VERSION} && pip3 install -e .
+
+
+# download additional embeddings
+RUN genienlp cache-embeddings -d /usr/local/share/genienlp/embeddings --embeddings bert-base-multilingual-uncased
```
Comment (gcampax) [inline Dockerfile]: This should probably be in the base image instead

File: Dockerfile [hunk 2]
```diff
@@ -23,10 +29,16 @@ RUN git checkout ${GENIE_VERSION}
...
-COPY lib.sh generate-dataset-job.sh train-job.sh evaluate-job.sh .
+COPY lib.sh generate-dataset-job.sh train-job.sh evaluate-job.sh ./
 
 # add user genie-toolkit
 RUN useradd -ms /bin/bash -r genie-toolkit
 USER genie-toolkit
 WORKDIR /home/genie-toolkit
 
+USER root
+RUN chmod -R 755 /usr/local/share/genienlp/embeddings
```
Comment (gcampax) [inline Dockerfile]: Same as this one really

File: train-job.sh [hunk 3]
```diff
@@ -20,9 +20,14 @@ ln -s "$HOME/dataset" "$modeldir/dataset/${task_name}"
 ln -s $modeldir /home/genie-toolkit/current
 mkdir -p "/shared/tensorboard/${experiment}/${owner}/${model}"
 
+echo '****'
+uname
+whoami
+
+
```
Comment (gcampax) [inline train-job.sh:28]: debugging code?

File: evaluate.sh [hunk 4]
```diff
@@ -8,7 +8,7 @@ shift $n
 check_config "IAM_ROLE OWNER IMAGE"
 
 JOB_NAME=${OWNER}-evaluate-${experiment}-${model}
-cmdline="--owner ${owner} --experiment $experiment --dataset $dataset --model $model --workdir ${workdir} -- "$(requote "$@")
+cmdline="--owner ${OWNER} --experiment $experiment --dataset $dataset --model $model --workdir ${workdir} -- "$(requote "$@")
```
Comment (gcampax) [inline evaluate.sh]: @rayslxu you also have some changes here, right?

**Review by gcampax — APPROVED** [review]
Ya, some comments

File: Dockerfile (hunk 1 above)
Comment (Mehrad0711) [reply Dockerfile]: will move it then.

File: Dockerfile (hunk 2 above)
Comment (Mehrad0711) [reply Dockerfile]: yeah. without this line user "genie-toolkit" could not read from that path...

File: train-job.sh (hunk 3 above)
Comment (Mehrad0711) [reply train-job.sh:28]: will delete it.

File: Dockerfile (hunk 2 above)
Comment (gcampax) [reply Dockerfile]: Yeah it just needs to be in the common image instead

File: evaluate.sh (hunk 4 above)
Comment (sileix) [reply evaluate.sh]: yup

**Pull request merged**